handbook/tools/social-media.md

```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
 
 ## Accounts
 
-You can find direct links to our accounts in the [`ubclaunchpad.com` site configuration](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/ubclaunchpad/ubclaunchpad%5C.com%24+file:config.ts+socials%28.*%3F%29%5C%7B%28.%7C%5Cn%29*%3F%7D&patternType=regexp&case=yes).
+You can find direct links to our accounts in the [`ubclaunchpad.com` site configuration](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/ubclaunchpad/ubclaunchpad%5C.com%24+file:config.ts+socials%28.*%3F%29%5C%7B%28.%7C%5Cn%29*%3F%7D&patternType=regexp&case=yes). Accounts can easily be linked via [redirects](https://ubclaunchpad.com/config/#adding-a-custom-redirect) (this can also be used to add short links for other things), for example:
 
-Credentials are available in the [Exec repository](https://github.com/ubclaunchpad/exec) - please reach out to [`#ask-leads`](https://ubclaunchpad.slack.com/archives/CK935RD3Q) to get in touch with one of the presidents if you are interested in helping us run our social media accounts!
+* [https://ubclaunchpad.com/facebook](https://ubclaunchpad.com/facebook)
+* [https://ubclaunchpad.com/instagram](https://ubclaunchpad.com/instagram)
+* [https://ubclaunchpad.com/medium](https://ubclaunchpad.com/medium)
 
-### Facebook
+Credentials for accounts are available in the [Exec repository](https://github.com/ubclaunchpad/exec) - please reach out to [`#ask-leads`](https://ubclaunchpad.slack.com/archives/CK935RD3Q) to get in touch with one of the presidents if you are interested in helping us run our social media accounts!
 
-TODO
+Please reach out to the presidents to create accounts for social media or third-party services so that we can keep track of them. Also see our [email documentation](./email.md).
```

Comment:
Just want to make sure that `email.md` isn't implemented as of yet?

Reply:
yep this should be fine, the email docs are still pending review over at https://github.com/ubclaunchpad/docs/pull/138